packages/database/lib/schema.ts

```diff
@@ -1,149 +1,215 @@
-  bidsTokenAcct: pubkey('bids_token_acct').references(() => tokenAccts.tokenAcct).notNull(),
-  asksTokenAcct: pubkey('asks_token_acct').references(() => tokenAccts.tokenAcct).notNull(),
+  bidsTokenAcct: pubkey("bids_token_acct")
+    .references(() => tokenAccts.tokenAcct)
+    .notNull(),
+  asksTokenAcct: pubkey("asks_token_acct")
+    .references(() => tokenAccts.tokenAcct)
+    .notNull(),
 
   // Fees are in bips
-  baseMakerFee: smallint('base_maker_fee').notNull(),
-  baseTakerFee: smallint('base_taker_fee').notNull(),
-  quoteMakerFee: smallint('quote_maker_fee').notNull(),
-  quoteTakerFee: smallint('quote_taker_fee').notNull(),
+  baseMakerFee: smallint("base_maker_fee").notNull(),
+  baseTakerFee: smallint("base_taker_fee").notNull(),
+  quoteMakerFee: smallint("quote_maker_fee").notNull(),
+  quoteTakerFee: smallint("quote_taker_fee").notNull(),
 
   // When market becomes active or inactive
-  activeSlot: slot('active_slot'),
-  inactiveSlot: slot('inactive_slot')
+  activeSlot: slot("active_slot"),
+  inactiveSlot: slot("inactive_slot"),
 });
 
-export const twaps = pgTable('twaps', {
-  marketAcct: pubkey('market_acct').references(() => markets.marketAcct).notNull(),
-  proposalAcct: pubkey('proposal_acct').references(() => proposals.proposalAcct).notNull(),
-  updatedSlot: slot('updated_slot').notNull(),
-  // max u128 value is 340282366920938463463374607431768211455 (39 digits)
-  // the account field is u128 https://github.com/metaDAOproject/openbook-twap/blob/82690c33a091b82e908843a14ad1a571dfba12b1/programs/openbook-twap/src/lib.rs#L52
-  observationAgg: numeric('observation_agg', {precision: 40, scale: 0}).notNull(),
-  curTwap: tokenAmount('token_amount').notNull(),
-}, table => ({
-  pk: primaryKey(table.marketAcct, table.updatedSlot)
-}));
-
-export const transactions = pgTable('transactions', {
-  txSig: transaction('tx_sig').primaryKey(),
-  slot: slot('slot').notNull(),
-  blockTime: timestamp('block_time').notNull(),
-  failed: boolean('failed').notNull(),
-  payload: text('payload').notNull(),
-  serializerLogicVersion: smallint('serializer_logic_version').notNull(),
-}, table => ({
-  slotIdx: index('txn_slot_index').on(table.slot)
-}));
+export const twaps = pgTable(
+  "twaps",
+  {
+    marketAcct: pubkey("market_acct")
+      .references(() => markets.marketAcct)
+      .notNull(),
+    proposalAcct: pubkey("proposal_acct")
+      .references(() => proposals.proposalAcct)
+      .notNull(),
+    updatedSlot: slot("updated_slot").notNull(),
+    // max u128 value is 340282366920938463463374607431768211455 (39 digits)
+    // the account field is u128 https://github.com/metaDAOproject/openbook-twap/blob/82690c33a091b82e908843a14ad1a571dfba12b1/programs/openbook-twap/src/lib.rs#L52
+    observationAgg: numeric("observation_agg", {
+      precision: 40,
+      scale: 0,
+    }).notNull(),
+    curTwap: tokenAmount("token_amount").notNull(),
+    createdAt: timestamp("created_at")
```

Comment:
Since this is default then you don't likely need to pass it in the actual insert in the function just FYI.